### ISSUE TITLE
Update Collections Display

### ISSUE THREAD

**Issue by allysonlister** [FAIRsharing/fairsharing.github.io#1323]
Collection records are allowed to have contacts, but although such contacts are editable within the edit interface, they are not being displayed within the record view and should be.

Example: https://beta.fairsharing.org/3607 has two contacts, neither of which are shown.

As a larger issue, please could you go through all valid collection fields and find out which are not being displayed in the record view? We need to start showing anything that is allowed within the data model.

Haven't created the ticket as a bug as it may be there was some constraint when the front end was created that I was unaware of.

**Comment by knirirr** [comment]
The specifications were changed during development so that the display elements where (for example) contacts would normally be displayed were suppressed in favour of code derived from the search page:

https://github.com/FAIRsharing/fairsharing.github.io/issues/912

Reverting collections' pages to the same as any other FairsharingRecord would cause the relevant items to be displayed. But, I suspect that this would not be considered acceptable as the record search results and facets would disappear.

So, before any work on this ticket (which could be fairly involved) proceeds, it would be useful to have a little more information on how you would expect a collection page to look. For example, would you want to see the contacts element above the list of records/facets, or below?

**Comment by allysonlister** [comment]
Thank you, I agree completely. Let me find out if SAS has any preferences and get back to you.

**Comment by allysonlister** [comment]
OK I have had a meeting with SAS about this and have created a specification at https://app.gitbook.com/o/-MchtBAOWXh2gO--zz7m/s/-MkSKpWh9NA-iywIqhP_/please-can-we/record-specific-specifications/collection-display-specification

Essentially, she does not want any more additional cards/elements than there are now, which means most elements should be hidden or perhaps removed from the edit interface (whichever is the better solution for you), other than these three: contacts, organisations, publications. In order for these three to display, she would like a simple text version of these elements displayed within the General Information card.

Please see the document I linked above and let me know if you would like a meeting / what kind of questions you have.

Thanks!

**Comment by allysonlister** [comment]
After further discussion, it was decided that the edit form will remain untouched, and these changes will only affect the record view. While users will be able to update other sections via the API and the edit interface, and these updates will be saved, they will not be visible in the record view. We do not anticipate them needing any additional fields, and if the users request any of them, we can open a new ticket if we think it is suitable.

**Comment by hoseinmirian** [comment]
> OK I have had a meeting with SAS about this and have created a specification at https://app.gitbook.com/o/-MchtBAOWXh2gO--zz7m/s/-MkSKpWh9NA-iywIqhP_/please-can-we/record-specific-specifications/collection-display-specification
> 
> Essentially, she does not want any more additional cards/elements than there are now, which means most elements should be hidden or perhaps removed from the edit interface (whichever is the better solution for you), other than these three: contacts, organisations, publications. In order for these three to display, she would like a simple text version of these elements displayed within the General Information card.
> 
> Please see the document I linked above and let me know if you would like a meeting / what kind of questions you have.
> 
> Thanks!

I cannot access the file. please give me permission @allysonlister

**Comment by allysonlister** [comment]
You need to have an account with gitbook - I'll send the invitation via zulip.
